install/windows.md

```diff
@@ -8,24 +8,35 @@ Wir empfehlen die Installation einer 64-bit Version, falls man ein 64-bit Betrie
 Wenn man sich nicht sicher ist, hat man heutzutage fast immer 64-bit.
 
 
-__Es müssen alle Schritte (Installieren, Testen, Aktualisieren) ausgeführt werden!__
+__Bitte die folgenden Punkte unbedingt lesen:__
 
-Falls man nicht am LaTeX-Kurs teilnimmt, ist TeXLive optional.
-Dann kann man allerdings kein TeX in matplotlib benutzen (führt zu weniger schönen Plots).
++ Diese Anleitung mag relativ lang erscheinen und im ersten Moment abschreckend wirken.
+Wir versichern euch, dass diese Länge aber viel mehr ein Zeichen von Ausführlichkeit
+ist, um sicherzustellen, dass ihr auch folgen könnt, wenn ihr bisher keine Berührungen
+mit den Inhalten des Toolbox Workshops hattet. 
 
-Falls man nur am LaTeX-Kurs teilnehmen will, sollte man mindestens Sumatra und VSCodium (oder einen anderen Texteditor) und TeXLive installieren.
++ Befolgt die Anleitung daher konzentriert und gründlich, um keinen Schritt zu vergessen
+und kontrolliert jedesmal, dass die ihr die Befehle richtig kopiert oder abgetippt habt.
+
++ Alle drei Abschnitt (Installieren, Testen, Aktualisieren) und deren Unterabschnitte sind notwendig.
+Den Beginn jedes dieser drei Abschnitte erreicht ihr über die drei Buttons unter der Einleitung. 
+
+
++ Sollte es an irgendeiner Stelle zu Problemen kommen, überspringt diesen Schritt nicht einfach,
+sondern meldet euch bei uns. Unter [Hilfe](/problem.html) findet ihr die Hinweise wie ihr uns 
+am besten kontaktieren könnt.
+
+
+Falls falls ihr __nicht__ am LaTeX-Kurs teilnehmen wollt, ist der Abschnitt <a href="#TeXLive">TeXLive </a> optional.
+Dann könnt ihr allerdings kein TeX in matplotlib benutzen (führt zu weniger schönen Plots).
+
+Falls ihr __nur__ am LaTeX-Kurs teilnehmen wollt, solltet ihr mindestens die Abschnitte <a href="#SumatraPDF">Sumatra PDF</a>, 
```

Comment:
mindestens die Unterabschnitte

Reply:
Thinking about it: I guess it woud be better to list of, which subsections can be skipped